api/net/Areas/Services/Controllers/ReportController.cs

```diff
@@ -93,11 +94,11 @@ public IActionResult FindById(int id)
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [ProducesResponseType((int)HttpStatusCode.NoContent)]
     [SwaggerOperation(Tags = new[] { "Report" })]
-    public async Task<IActionResult> FindContentForReportIdAsync(int id, int? requestorId)
+    public async Task<IActionResult> FindContentForReportIdAsync(int id, long? instanceId, int? requestorId)
```

Comment:
Need to pass the instance in some cases so that it is used to exclude content when making a search for more.